packages/safe-tools-client/app/components/future-payments-list/index.gts

```diff
@@ -111,9 +124,10 @@ export default class FuturePaymentsList extends Component<Signature> {
         <Section @title="Future Payments" data-test-future-payments-list>
           <div class="future-payments-list__payments-section">
             <div class="future-payments-list__payments-section-time-brackets">
-              <ScheduledPaymentTimeBracket @title="next hour" @scheduledPayments={{this.nextHour}}/>
-              <ScheduledPaymentTimeBracket @title="next month" @scheduledPayments={{this.nextMonth}}/>
-              <ScheduledPaymentTimeBracket @title="next few months" @scheduledPayments={{this.nextFewMonthsPayments}}/>
+              <ScheduledPaymentTimeBracket @title="today" @scheduledPayments={{this.today}}/>
+              <ScheduledPaymentTimeBracket @title="tomorrow" @scheduledPayments={{this.tomorrow}}/>
+              <ScheduledPaymentTimeBracket @title="this month" @scheduledPayments={{this.thisMonth}}/>
+              <ScheduledPaymentTimeBracket @title="later" @scheduledPayments={{this.later}}/>
```

Comment:
Will this display "Later" when all scheduled payments happen next month?

Reply:
Yes, in that case, the "Later" time window will be displayed.

Reply:
I think showing "Later" string doesn't make sense when then there are no earlier other time windows displayed

Reply:
Do you mean to hide all scheduled payments or change the label?

Reply:
Updated